assets/components/infoSection/infoSection.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import type { Children } from 'react';
+
+
+// ---- Types ----- //
+
+type PropTypes = {
+  heading?: string,
+  className?: string,
+  children?: Children,
+};
+
+
+// ----- Component ----- //
+
+export default function InfoSection(props: PropTypes) {
+
+  const headerContent = (
+    <h2 className="component-info-section__heading">{props.heading}</h2>
```

Comment:
If nothing is passed through as the heading, React renders... nothing. It just treats null as an empty string I believe.